specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/LabelHistory_Get.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppName": "testcontainerApp0",
+    "labelName": "dev",
+    "api-version": "2024-10-02-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+          "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/containerApps/testcontainerApp0/labelHistory/dev",
+          "name": "dev",
+          "type": "Microsoft.App/containerApps/labelHistory",
+          "properties": {
+            "label": "dev",
```

Comment:
if `name` property already has label name, this seems redundant

Reply:
the name here should have been the containerapp name fixed it